app/src/main/java/studyBuddy/SessionCompleteCallback.java

```diff
@@ -1,5 +1,5 @@
 package studyBuddy;
 
 public interface SessionCompleteCallback {
-    void callbackFunc();
+    void callbackFunc(long elapsedTime);
```

Comment:
This function accepts a long now but the callback in TimerRunner has not been passed the parameter